### DISCUSSION TITLE
Welcome to rapyd_sdk_flutter Discussions!

### DISCUSSION THREAD

**Discussion by sbis04** [sbis04/rapyd_sdk_flutter#1] [Announcements]
## 👋  Hey, welcome to Rapyd Flutter SDK discussion!

This package is in a very early stage, just has a very few API endpoints that I needed at the time of working on a project. That being said, it has everything required for performing proper API calls (including salt and signature generation), which I found many people were struggling with.

Package Link: https://pub.dev/packages/rapyd

You can use this space for any discussion regarding this package and if you want to contribute to it. Currently, the package lacks good structuring, which would be a good place to start with. Feel free to raise any PRs.

Looking forward to your suggestions! 💙

**Comment by iNeedHelpX** [comment]
thank you very much for doing this for us!!
